Automattic/jetpack
# Boost: Add Image CDN quality control

Fixes Automattic/boost-cloud#364

## Proposed changes:
* Added a quality slider for premium users.
* Registered a new data-sync entry to check premium features from front-end.

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:
* Apply the filter `add_filter( 'jetpack_boost_has_feature_image-cdn-quality', '__return_true' );` to enable the feature
* Check Boost settings to see if the quality slider is visible, and that changing the slider changes the image quality.
* While the slider is not at 100, remove the filter to disable the feature.
* Check the image quality to make sure they are at 100%.

![image](https://github.com/Automattic/jetpack/assets/3737780/58bdc5fa-ab0d-4ee7-a573-be07dec4e77e)
